custom_components/tuya_local/devices/alecoair_pu55_humino.yaml

```diff
@@ -0,0 +1,180 @@
+name: PU55 Humino
+products:
+  - id: rjhsz1vaxpp4gih3
+    manufacturer: AlecoAir
+    model: PU55 HUMINO
+
+entities:
+  - entity: humidifier
+    class: humidifier
+    translation_key: humidifier_with_presets
+    name: Humidifier
+    dps:
+      - id: 105
+        name: switch
+        type: boolean
+      - id: 101
+        name: humidity
+        type: integer
+        range:
+          min: 30
+          max: 70
+        mapping:
+          - step: 5
+      - id: 13
+        name: current_humidity
+        type: integer
+
+  - entity: fan
+    translation_key: fan_with_presets
+    name: Purifier
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+      - id: 4
+        name: speed
+        type: string
+        mapping:
+          - dps_val: low
+            value: 25
+          - dps_val: mid
+            value: 50
+          - dps_val: high
+            value: 75
+          - dps_val: max
+            value: 100
+      - id: 3
+        name: preset_mode
+        type: string
+        mapping:
+          - dps_val: normal
+            value: normal
+          - dps_val: ai_clean
+            value: AI Clean
+          - dps_val: sleep
+            value: sleep
+
+  - entity: sensor
+    name: Filter Life
+    dps:
+      - id: 5
+        name: sensor
+        type: integer
+        unit: "%"
+        class: measurement
+
+  - entity: sensor
+    name: Humidity
+    class: humidity
+    dps:
+      - id: 13
+        name: sensor
+        type: integer
+        unit: "%"
+        class: measurement
+
+  - entity: sensor
+    name: Air Quality
+    dps:
+      - id: 21
+        name: sensor
+        type: string
+
+  - entity: sensor
+    name: Clean Water Tank
```

Comment:
HA naming convention is to only capitalise the first word in an entity name (with exceptions for abbreviations)